app/townsquare/templates/townsquare/index.html

```diff
@@ -127,6 +127,7 @@ <h4 class="pt-3">{{type|title}} Action</h4>
 <link rel="stylesheet" href="{% static "v2/css/activity_stream.css" %}">
 <link rel="stylesheet" href="{% static "v2/css/town_square.css" %}">
 <script src="{% static "v2/js/status.js" %}"></script>
+<script src="{% static 'v2/js/lib/emoji-button.js' %}"></script>
```

Comment:
youll want to add this to activty_scripts.html not to townsquare/index.html because the status box is shown in a few diff places not just townsquare/index !!!